code/modules/long_range_scanner/long_range_scanner.dm

```diff
@@ -0,0 +1,482 @@
+#define EVENT_ENABLED 			3
+#define EVENT_DISABLED 			4
+#define EVENT_RECONFIGURED		5
+#define PASSIVE_SCAN_RANGE      3
+#define PASSIVE_SCAN_PERIOD     3 SECONDS
```

Comment:
Why don't the values have the same distance
![emoji](https://user-images.githubusercontent.com/61743710/112637973-0a310280-8e3f-11eb-9fe1-3c5e9c6038b3.png)

Reply:
There are aligned in Visual Studio Code ;-;